deployer/templates/heapster-standalone.yaml

```diff
@@ -75,9 +77,9 @@ objects:
           - "heapster-wrapper.sh"
           - "--wrapper.allowed_users_file=/secrets/heapster.allowed-users"
           - "--source=kubernetes:${MASTER_URL}?useServiceAccount=true&kubeletHttps=true&kubeletPort=10250"
-          - "--tls_cert=/secrets/heapster.cert"
-          - "--tls_key=/secrets/heapster.key"
-          - "--tls_client_ca=/secrets/heapster.client-ca"
+          - "--tls_cert=/heapster-certs/tls.crt"
+          - "--tls_key=/heapster-certs/tls.key"
+          - "--tls_client_ca=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
```

Comment:
This is the wrong ca, its needs to the main ca, not the service-ca.crt
eg: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"